Engine_Revit_UI/Engine_Revit_UI.csproj

```diff
@@ -169,6 +169,10 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\..\BHoM_Engine\Build\Reflection_Engine.dll</HintPath>
     </Reference>
+    <Reference Include="Reflection_oM, Version=3.0.0.0, Culture=neutral, processorArchitecture=MSIL">
```

Comment:
Version, Culture, and Architecture should be removed

Reply:
Fixed.